app/(dashboard)/dashboard/_components/SummaryStatistics/SummaryStatistics.tsx

```diff
@@ -1,56 +1,78 @@
+'use client';
+
+import Image from 'next/image';
 import Link from 'next/link';
 import ResponsiveContainer from '~/components/ResponsiveContainer';
-import StatCard from './StatCard';
-import Image from 'next/image';
+import { api } from '~/trpc/client';
 import { InterviewIcon, ProtocolIcon } from './Icons';
-import { api } from '~/trpc/server';
-import { unstable_noStore } from 'next/cache';
+import StatCard, { StatCardSkeleton } from './StatCard';
 
-export const dynamic = 'force-dynamic';
+export default function SummaryStatistics() {
+  const { data: protocolCount, isLoading: isProtocolStatsLoading } =
+    api.dashboard.getSummaryStatistics.protocolCount.useQuery();
 
-export default async function SummaryStatistics() {
-  unstable_noStore();
+  const { data: participantCount, isLoading: isParticipantStatsLoading } =
+    api.dashboard.getSummaryStatistics.participantCount.useQuery();
 
-  const interviewCount =
-    await api.dashboard.getSummaryStatistics.interviewCount.query();
-  const participantCount =
-    await api.dashboard.getSummaryStatistics.participantCount.query();
-  const protocolCount =
-    await api.dashboard.getSummaryStatistics.protocolCount.query();
+  const { data: interviewCount, isLoading: isInterviewStatsLoading } =
+    api.dashboard.getSummaryStatistics.interviewCount.useQuery();
 
   return (
     <ResponsiveContainer
       className="grid grid-cols-1 gap-4 sm:grid-cols-3 lg:gap-6"
       maxWidth="6xl"
     >
       <Link href="/dashboard/protocols">
-        <StatCard
-          title="Protocols"
-          initialData={protocolCount}
-          icon={<ProtocolIcon />}
-        />
+        {isProtocolStatsLoading ? (
+          <StatCardSkeleton title="Protocols" icon={<ProtocolIcon />} />
+        ) : (
+          <StatCard
+            title="Protocols"
+            initialData={protocolCount!}
```

Comment:
Let's rename this prop to just `data` or `value` now that it isn't being used as initialData.

Reply:
Done here: https://github.com/complexdatacollective/Fresco/pull/115/commits/518153d7287f254ccaa45993d9e566b8ed8496a6